src/rdb.h

```diff
@@ -13,7 +13,7 @@ void Discard_Globals_Backup();
 
 // For rdb short read
 
-#define LoadStringBufferAlloc_IOErrors(rdb, ptr, len, cleanup_exp)  \
+#define LoadStringBufferAlloc_IOErrors(rdb, ptr, len, exclude_null_delimiter_from_len, cleanup_exp)  \
```

Comment:
Bug fix?
I see it is called with `true` always, so why do we need it as an argument?
Please explain the addition (in this case substraction 😉)

Reply:
The length that is returned includes the null delimiter.
HiddenName holds a length which is supposed to be equivalent to strlen()
Wanted to avoid using strlen so wanted to use the length that was returned here.
Made it optional to accomodate other future users.
I think this macro got later modified in the rlookup PR.
I can backport the changes from that PR into this one.
Let me know what you prefer.